tetris.py

```diff
@@ -407,89 +467,65 @@ def flasher(self, full_lines, fill):
         """
         for x in range(game_width):
             for x_line in full_lines:
-                self.draw_rectangle(x, x_line, "flash", fill)
-
+                self.main_board.draw_rectangle(x, x_line, "flash", fill)
 
-class TetrisGUI:
-    def __init__(self, topbar_time, game_canvas, topbar_score, topbar_canvas):
-        self.topbar_time = topbar_time
-        self.topbar_canvas = topbar_canvas
-        self.game_canvas = game_canvas
-        self.topbar_score = topbar_score
-        self.new_game()
+    def timer(self):
+        if self.game_status == GameStatus.in_progress:
+            game_time = time.time() - self.start_time - self.paused_time
+            self.topbar_time.config(text=f"{int(game_time / 60):02d}:{int(game_time % 60):02d}")
+            self.topbar_time.after(1000, self.timer)
 
-    def game_over_check(self):
-        y_coordinates = [y for (x, y) in self.main_board.coord_extractor()]
+    def game_over(self, forced=False):
```

Comment:
There are two `def game_over(...)` in this code, this one here in `class Game`, and another in `class TetrisControl`. The one in `Game` is never used. Please delete it.